src/sites_toolbox.py

```diff
@@ -28,6 +30,36 @@ def get_user_details(session, email):
     ]
     return user_sites, user_site_group, user_site_count
 
+# get details for one site
+
+def get_site_details(session, site_selection):
+  """Get the site details for one site"""
```

Comment:
should `site_selectionz bu `site_uuid`

Reply:
yeah, it makes sense to do site_uuid since I don't actually pass the client_site_id into it.